app/Http/Controllers/Service/Admin/TradersController.php

```diff
@@ -259,6 +259,8 @@ public function download()
         $traders = $this->getTraders();
         $csvExporter = new Export();
         $csvExporter->beforeEach(function ($trader) {
+            $trader->joined = $trader->created_at->format('d-m-Y');
+            $trader->disabled = $trader->disabled_at ? $trader->disabled_at->format('d-m-Y') : $trader->disabled_at;
```

Comment:
Is the behaviour of "null" to print nothing in the cell?

Reply:
yes, `null` prints nothing in the cell. also, 'null' just explodes the whole thing - there's no method `format` on `null` - hence the ternary.